som_polissa/som_polissa_webforms_helpers.py

```diff
@@ -0,0 +1,65 @@
+from osv import osv
+
+class SomPolissaWebformsHelpers(osv.osv_memory):
+
+    _name = 'som.polissa.webforms.helpers'
+
+    def www_get_iban(self, cursor, uid, ids, context=None):
+        if context is None:
+            context = {}
+        if isinstance(ids, (list, set, tuple)):
+            polissa_id = ids[0]
+        else:
+            polissa_id = ids
+
+        pol_obj = self.pool.get('giscedata.polissa')
+        bank = pol_obj.read(
+            cursor, uid, polissa_id, ['bank'], context=context
+        )
+        iban = ''
+        if bank.get('bank'):
+            iban_prefix = bank['bank'][1][-4:]
+            iban = '**** **** **** **** **** {}'.format(iban_prefix)
+        return iban
+
+    def www_check_iban(self, cursor, uid, iban, context=None):
+        if context is None:
+            context = {}
+
+        pol_obj = self.pool.get('giscedata.polissa')
+        result = pol_obj.www_check_iban(cursor, uid, iban, context=context)
+
+        return result
+
+    def www_set_iban(self, cursor, uid, ids, iban, context=None):
+        if context is None:
+            context = {}
+        if isinstance(ids, (list, set, tuple)):
```

Comment:
Maybe would be worth follow the rule of three and apply a semantic abstraction for this check if we think could be fine to improve readability